<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bootleg NFO Builder for Plex</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="ShowInfoForm">
      <item row="0" column="0">
       <widget class="QLabel" name="showNameLabel">
        <property name="text">
         <string>Show Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="showNameInput">
        <property name="placeholderText">
         <string>The shows name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="showDateLabel">
        <property name="text">
         <string>Performance Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="showDateInput">
        <property name="placeholderText">
         <string>YYYY-MM-DD</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="showLocationLabel">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="showLocationInput">
        <property name="placeholderText">
         <string>Location/Studio</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="showTaglineLabel">
        <property name="text">
         <string>Tag Line</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="showTaglineInput">
        <property name="placeholderText">
         <string>A tagline for the show</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="showSynopsisLabel">
        <property name="text">
         <string>Synopsis</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="showSynopsisInput">
        <property name="placeholderText">
         <string>A brief synopsis of the show, or notes</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="showPosterLabel">
        <property name="text">
         <string>Show Poster URL</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="showPosterInput">
        <property name="placeholderText">
         <string>Full URL to image to use as Poster</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="CastLabel">
      <property name="text">
       <string>Cast Members</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="castTable">
      <property name="rowCount">
       <number>20</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Actor/Actress Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Character Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Headshot URL</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextArea">
      <property name="placeholderText">
       <string>Output will be here.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CreateNFO">
      <property name="text">
       <string>Create NFO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
